actors/common/src/main/java/org/sunbird/metrics/actors/CourseMetricsActor.java

```diff
@@ -204,7 +214,8 @@ private void courseProgressMetrics(Request actorMessage) {
         ElasticSearchUtil.getDataByIdentifier(
             EsIndex.sunbird.getIndexName(), EsType.course.getTypeName(), batchId);
     if (isNull(courseBatchResult) || courseBatchResult.size() == 0) {
-      ProjectLogger.log("CourseMetricsActor-courseProgressMetrics-- batch is not valid .");
+      ProjectLogger.log(
+          "CourseMetricsActor:courseProgressMetrics  batch is not valid .", LoggerEnum.INFO.name());
```

Comment:
From:
"CourseMetricsActor:courseProgressMetrics  batch is not valid ."

To:
"CourseMetricsActor:courseProgressMetrics: batchId not found."